.travis.yml

```diff
@@ -8,7 +8,6 @@ env:
   global:
     - ARCH=$(go env GOARCH)
     - TAG="ci"
-    - CODECOV_TOKEN="d1e39bea-f800-45de-8266-3e9b85e2594a"
```

Comment:
Is there a new token being used now ?

Reply:
Nope. As per codecov actions [README](https://github.com/codecov/codecov-action/blame/master/README.md#L12), public projects no longer require a token for providing coverage report.